README.md

```diff
@@ -5,6 +5,16 @@
 This repository is for the OpenMRS Dispensing App. For more information, please see the
 [Design Documentation](https://wiki.openmrs.org/display/projects/Dispensing+Design+Components).
 
+## Required privileges
+
+Note that following privileges need to be installed and assigned to roles:
+
+* o3.dispensing-app.dispense.create: Allows user to Dispense Medication
+* o3.dispense-app.dispense.create.modifyDetails: Allows user to modify the Quantity, Drug, Formulation and Dose Instructions (from the values specified in the Order / Medication Request) when Dispensing
```

Comment:
dispense-app or dispensing-app?  Do we really need the o3 prefix?

Reply:
Oh, nice catch, I did this while waiting to board my red-eye and looks like I got sloppy and missed this, will correct.

I'll comment about the "o3" prefix on the other thread....